src/containers/Card.js

```diff
@@ -104,16 +66,63 @@ const Card = props => {
   );
 };
 
+const renderTitle = (title, image, titleStyle, fontFamily, dividerStyle) => {
+  if (!title) {
+    return null;
+  }
+
+  if (typeof title !== 'object') {
+    title = (
+      <Text
+        style={[
+          styles.cardTitle,
+          image && styles.imageCardTitle,
+          titleStyle && titleStyle,
+          fontFamily && {fontFamily},
+        ]}
+      >
+        {title}
+      </Text>
+    );
+  }
+
+  return (
+    <View>
+      {title}
+      {!image &&
+      <Divider
+        style={[styles.divider, dividerStyle && dividerStyle]}
+      />}
+    </View>
+  );
+};
+
+const renderFeaturedTitle = (title, titleStyle, defaultStyle) => {
```

Comment:
Consider renaming this to something more generic such as `renderFeatureText` or something similar. It's doing more than just `renderFeaturedTitle`